docs/source/swift-scripting.md

```diff
@@ -203,10 +187,10 @@ This is best achieved with a Run Script Build Phase.
 
     ```
     cd "${SRCROOT}"/Codegen
-    xcrun -sdk macosx swift run
+    xcrun -sdk macosx swift run generate
     ```
 
-    >**Note**: If your package ever seems to have problems with caching, run `swift package clean` before `swift run` for a totally clean build. It is not recommended to do this by default, because it substantially increases build time.
+    >**Note**: If your package ever seems to have problems with caching, run `swift package clean` before `swift run` for a totally clean build. Do not this by default, because it substantially increases build time.
```

Comment:
"Do not this by default"? Getting to the end of the day? ;)

Reply:
i english gud